packages/story-editor/src/components/dropTargets/provider.js

```diff
@@ -203,20 +205,25 @@ function DropTargetsProvider({ children }) {
     [activeDropTargetId, combineElements, elements, dropTargets, pushTransform]
   );
 
+  // ⌘ key disables drop-targeting.
+  const isDropTargetingDisabled = useGlobalIsKeyPressed('meta');
+
   const state = {
     state: {
       dropTargets,
       activeDropTargetId,
       draggingResource,
+      isDropTargetingDisabled,
     },
+    // If drop-targeting is disabled, all the related actions are ignored.
     actions: {
-      registerDropTarget,
+      registerDropTarget: isDropTargetingDisabled ? noop : registerDropTarget,
       unregisterDropTarget,
-      isDropSource,
-      isDropTarget,
-      handleDrag,
-      handleDrop,
-      setDraggingResource,
+      isDropSource: isDropTargetingDisabled ? () => false : isDropSource,
```

Comment:
Since drop-targeting actions are used in various places, the easiest way seemed to be just disabling these directly from the provider instead of changed each usage separately + tracking potential future cases. Doesn't look the cleanest but seemed to be the cleanest approach. LMK in case of alternative ideas.

Reply:
Hmm, it does the job very efficiently. But it's a bit dirty and non-transparent, but I also cannot come up with a cleaner solution.

Reply:
One minor "bug" though:

If you start dragging without pressing mod, hover a drop target and observe the potential drop, then press mod, then drag around more, the item is stuck to the drop target and can't "get out" until you release mod and the press mod again. But I'm not sure it's worth fixing, as it's so easy to get around.

Reply:
>  But I'm not sure it's worth fixing, as it's so easy to get around.

Hm, I'm not even certain what should happen in this case. Probably it's an edge case that we can address if it gets reported.